step2_기능명세서.md

```
@@ -0,0 +1,105 @@
+# step2 기능 명세서
+
+### 구현 흐름
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)